packages/checkinstall.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Checkinstall < Package
+  description 'CheckInstall is a computer program for Unix-like operating systems which eases the installation and uninstallation of software compiled from source by making use of package management systems.'
+  homepage 'http://asic-linux.com.mx/~izto/checkinstall/'
+  version '1.6.2'
+  source_url 'http://asic-linux.com.mx/~izto/checkinstall/files/source/checkinstall-1.6.2.tar.gz'
+  source_sha256 'dc61192cf7b8286d42c44abae6cf594ee52eafc08bfad0bea9d434b73dd593f4'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  def self.patch
+    system 'wget', 'https://github.com/JL2210/patches/raw/f2da9fb13545311e67decef27626c45e4c6583e1/checkinstall-1.6.2-chromebrew.patch'
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read('checkinstall-1.6.2-chromebrew.patch') ) == '17706ab65698b7baa4cb2c1eda1c9ef3a41453069f6d83843f3c134439d3096a'
+    system 'patch -p1 < checkinstall-1.6.2-chromebrew.patch'
```

Comment:
Instead of trying to reinvent the wheel, why not just execute `crew build <pkgname>` when they select Chromebrew?  Also, you should be using `/usr/local/tmp` for the temp directory instead of `/usr/local/var/tmp` or it will crash and burn on CloudReady.

Reply:
How badly non-FHS compliant is Cloudready!?

Reply:
Also, that defeats the total purpose of Checkinstall.

Reply:
I don't think you understand how Chromebrew packages are built.  Take some time and examine how the build function works in the crew command.  `/usr/local/var` in CloudReady is a symlink to the read-only `/var` directory.

Reply:
`/var/tmp` is supposed to be mounted as tmpfs. When's the last time you used Cloudready? Crew build only accepts the name of a package, not a directory. I could try `crew build .` and all I would get back would be an error: `.: package not found :/`

Reply:
```
Loading extra module: /usr/share/crosh/dev.d/50-crosh.sh
shWelcome to crosh, the Chrome OS developer shell.

If you got here by mistake, don't panic!  Just close this tab and carry on.

Type 'help' for a list of commands.

If you want to customize the look/behavior, you can use the options page.
Load it by using the Ctrl+Shift+P keyboard shortcut.

crosh> shell
crewchronos@localhost / $ crew build .
Package . not found. :(
chronos@localhost / $ crew build /home/chronos/user/Downloads
Package /home/chronos/user/Downloads not found. :(
chronos@localhost / $ cd
chronos@localhost ~ $ cd Downloads
chronos@localhost ~/Downloads $ ls
Markdown.txt  Takeout  ada.rb  ada_bootstrap.rb  chroot  misc  patches  takeout-20180912T133929Z-001.tgz  update-cross-script-1
chronos@localhost ~/Downloads $ cd misc/
.clang-format               BLFS-BOOK-8.2-systemd.html  crew-build/                 uCoreutils/                 
.gitignore                  LFS-BOOK-8.2-systemd.pdf    hello-chromeos-gles2/       xfce_fat-4.12.tar.lz4       
chronos@localhost ~/Downloads $ cd misc/
chronos@localhost ~/Downloads/misc $ crew build uCoreutils                  
Package uCoreutils not found. :(
chronos@localhost ~/Downloads/misc $ 
```